.github/workflows/ci.yml

```diff
@@ -151,7 +151,7 @@ jobs:
       - name: Set up Python 3.10
```

Comment:
We should update this step name for consistency – or rather remove the specific Python version to avoid future Renovate update inconsistencies:

```suggestion
      - name: Set up Python
```